### PR TITLE
Cherry-pick docs for 0.4.19 release 

### PR DESCRIPTION
**Pull request by Czaki**
# Description

Cherrypic 22 commits from napari/docs repository

### CODE REVIEW

**Comment by Carreau** [comment]
restarting stuck test (qtdims2)

**Comment by psobolewskiPhD** [comment]
Last two have been merged: https://github.com/napari/docs/pulls?q=is%3Apr+sort%3Aupdated-desc+milestone%3A0.4.19+is%3Aclosed

Just a question of the multi canvas NAP

**Comment by Czaki** [comment]
aded docs PR

**Comment by psobolewskiPhD** [comment]
Could you please add:
https://github.com/napari/docs/pull/233
And, more importantly, https://github.com/napari/docs/pull/202
(conda install instructions fix)

And finally:
https://github.com/napari/docs/pull/157

File: docs/tutorials/fundamentals/viewer.md [hunk 1]
```diff
@@ -94,54 +94,54 @@ The viewer is organized into a few key areas which are explained in the next sec
...
 
-The layer list contains one widget for each of the layers that have been added to the viewer and includes a `thumbnail` that shows a miniaturized version of the currently viewed data, a `name` that is an editable text box, a `visibility` button that can be toggled on or off to show or hide the layer, and an `icon` for the layer type.  
+The layer list contains one widget for each of the layers that have been added to the viewer and includes a `thumbnail` that shows a miniaturized version of the currently viewed data, a `name` that is an editable text box, a `visibility` button (eye icon) that can be toggled on or off to show or hide the layer, and an `icon` for the layer type. Note that you can Option/Alt-click on the `visibility` button to show *just* that one layer, hiding all others. If you then Option/Alt-click on the `visibility` button of a layer a second time, the visibility state of all layers will be restored.
```
Comment (psobolewskiPhD) [inline docs/tutorials/fundamentals/viewer.md]: Not sure how this got here, the PR is flagged 0.5.0 (that's when the feature is to be released)
https://github.com/napari/docs/pull/121/files

**Comment by psobolewskiPhD** [comment]
Ugh, the white space changes make the diffs horrible 😭

File: docs/tutorials/fundamentals/viewer.md (hunk 1 above)
Comment (Czaki) [reply docs/tutorials/fundamentals/viewer.md]: it comes from https://github.com/napari/docs/pull/196/files#diff-648f35eb5f85b8494d2fa4e6a0d29edaaa329a5e89d1788415e39eb9d74c2d39

Comment (psobolewskiPhD) [reply docs/tutorials/fundamentals/viewer.md]: Ah, so David's auto formatter—I assume—tagged that paragraph which pulled in the whole thing and I didn't notice/realize when approving.

Comment (Czaki) [reply docs/tutorials/fundamentals/viewer.md]: Should I revert the whole changes in #196 in this file?

Comment (Czaki) [reply docs/tutorials/fundamentals/viewer.md]: Ok. It needs to be reverted manually.

Comment (psobolewskiPhD) [reply docs/tutorials/fundamentals/viewer.md]: The actual content (few headings changed) is so minor, so I'd be fine with it.
Documenting a feature that's not in the release seems bad.

Comment (Czaki) [inline docs/tutorials/fundamentals/viewer.md]:
```suggestion
The layer list contains one widget for each of the layers that have been added to the viewer and includes a `thumbnail` that shows a miniaturized version of the currently viewed data, a `name` that is an editable text box, a `visibility` button (eye icon) that can be toggled on or off to show or hide the layer, and an `icon` for the layer type.
```

This?

Comment (psobolewskiPhD) [inline docs/tutorials/fundamentals/viewer.md]:
```suggestion
The layer list contains one widget for each of the layers that have been added to the viewer and includes a `thumbnail` that shows a miniaturized version of the currently viewed data, a `name` that is an editable text box, a `visibility` button (eye icon) that can be toggled on or off to show or hide the layer, and an `icon` for the layer type.
```

alt-click feature isn't till 0.5.0

**Comment by psobolewskiPhD** [comment]
So the artifact from this has the wrong version for napari, e.g. in the installation docs, probably because the action is different?
Looking here:
https://github.com/napari/napari/blob/b4312d9516bc9e602c53bbb72317851052857009/.github/workflows/build_docs.yml#L20-L23

I think it needs `          fetch-depth: 0`

At the same time, I notice that in the other repos it shows 0.5.0 in the PR actions--latest tag I guess?

File: docs/naps/8-telemetry.md [hunk 2]
```diff
@@ -0,0 +1,251 @@
+:orphan: 
+
```
Comment (psobolewskiPhD) [inline docs/naps/8-telemetry.md]:
```suggestion  
```

Delete?

**Review by psobolewskiPhD — APPROVED** [review]

File: docs/naps/8-telemetry.md (hunk 2 above)
Comment (Czaki) [reply docs/naps/8-telemetry.md]: This is present also in main repository. So maybe PR to docs repo and cherry pick such PR?

Comment (psobolewskiPhD) [reply docs/naps/8-telemetry.md]: Done, approved, and marked ready-to-merge:
https://github.com/napari/docs/pull/269
So merge and cherry-pick at your leisure

**Comment by psobolewskiPhD** [comment]
@Czaki if it's not too much trouble, this one would be nice to have too:
https://github.com/napari/docs/pull/274
But it's pretty cosmetic, so it can wait.
❤️

**Comment by Czaki** [comment]
@psobolewskiPhD  #193 and #274 cherry picked

**Comment by psobolewskiPhD** [comment]
@Czaki I think you will also need:
https://github.com/napari/napari/pull/6547
in order to fix the glitch on the magicgui docs:
image

**Comment by Czaki** [comment]
It needs to be merged first

**Comment by psobolewskiPhD** [comment]
@Czaki here's the points annotation docs PR that would be nice to have in 0.4.19 before the stuff gets deprecated.
https://github.com/napari/docs/pull/145

**Pull request merged**